tests/suites/controller/unregister.sh

```diff
@@ -12,8 +12,12 @@ run_unregister() {
 	juju controllers --format=json | jq -r ".\"controllers\" | has(\"${controller_name}\")" | check true
 
 	echo "Backup controller info before unregister"
-	cp ~/.local/share/juju/controllers.yaml ~/.local/share/juju/controllers.yaml.bak
-	cp ~/.local/share/juju/accounts.yaml ~/.local/share/juju/accounts.yaml.bak
+	if [[ -f "${HOME}/.local/share/juju/controllers.yaml" ]]; then
```

Comment:
This appears wrong - if there's no controllers.yaml file, then there's no controller to unregister in the first place. Adding the file check seems to be hiding a genuine problem with the test. This is supported by the line abobe this

`juju controllers --format=json | jq -r ".\"controllers\" | has(\"${controller_name}\")" | check true`

Why are we using `| true` which can hide the fact that the expected controller doesn't exist. We should fail the test at that point if the juju controllers command fails.